ts/src/ellipx.ts

```diff
@@ -0,0 +1,1583 @@
+        return this.parseMarkets (markets);
+    }
+
+    parseMarket (market: Dict): Market {
+        const id = this.safeString (market, 'Market__');
+        const base = this.safeString (market['Primary'], 'Key');
+        const quote = this.safeString (market['Secondary'], 'Key');
+        const status = this.safeString (market, 'Status') === 'active';
+        const created = this.safeTimestamp (market['Created'], 'unix');
+        const amountPrecision = this.parseNumber (this.parsePrecision (this.safeString (market['Primary'], 'Decimals')));
+        const pricePrecision = this.parseNumber (this.parsePrecision (this.safeString (market['Secondary'], 'Decimals')));
+        return this.safeMarketStructure ({
+            'id': id,
+            'symbol': base + '/' + quote,
+            'base': base,
+            'quote': quote,
+            'settle': undefined,
+            'baseId': this.safeString (market['Primary'], 'Crypto_Token__'),
+            'quoteId': this.safeString (market['Secondary'], 'Crypto_Token__'),
+            'settleId': undefined,
+            'type': 'spot',
+            'spot': true,
+            'margin': false,
+            'swap': false,
+            'future': false,
+            'option': false,
+            'active': status,
+            'contract': false,
+            'linear': undefined,
+            'inverse': undefined,
+            'contractSize': undefined,
+            'expiry': undefined,
+            'expiryDatetime': undefined,
+            'strike': undefined,
+            'optionType': undefined,
+            'precision': {
+                'amount': amountPrecision,
+                'price': pricePrecision,
+            },
+            'limits': {
+                'amount': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+                'price': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+                'cost': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+            },
+            'info': market,
+            'created': created,
+        });
+    }
+
+    /**
+     * @method
+     * @name ellipx#fetchTicker
+     * @description fetches a price ticker, a statistical calculation with the information calculated over the past 24 hours for a specific market
+     * @param {string} symbol unified symbol of the market to fetch the ticker for
+     * @param {object} [params] extra parameters specific to the exchange API endpoint
+     * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/#/?id=ticker-structure}
+     */
+    async fetchTicker (symbol: string, params = {}): Promise<Ticker> {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const marketSymbol = market['symbol'].replace ('/', '_'); // Convert BTC/USDC to BTC_USDC
```

Comment:
Instead of doing these transformations we should use `market['id']` that's what is meant for (market['id'] should hold the exchange-specific market-id and market['symbol'] the ccxt-unified symbol)

Reply:
Change `market['id']` in `fetchMarkets` to resolve this.